data/json/items/gun/robofac_gun.json

```diff
@@ -424,5 +424,23 @@
     "ammo_type": [ "223" ],
     "flags": [ "MAG_BULKY" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "223": 100 } } ]
+  },
+  {
+    "id": "robofac60_123ln",
```

Comment:
You need to add a migration for this one to the generic guns mod. Should be good for merging after that